frontend/javascripts/oxalis/model/accessors/tool_accessor.ts

```diff
@@ -236,6 +242,7 @@ export function getDisabledInfoForTools(state: OxalisState): Record<
     hasAgglomerateMappings,
     genericDisabledExplanation,
     state.activeOrganization,
+    enforceActiveUser(state.activeUser),
```

Comment:
why enforce? `getFeatureNotAvailableInPlanMessage` should be able to handle null for the active user. i think this could crash currently when viewing a public annotation.